starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -51,8 +51,8 @@ public List<Inventory> findAll() {
    * @return Created/Updated Inventory.
    */
   public Inventory create(Inventory inventory) {
-    // TODO
-    return null;
+    this.mongoTemplate.save(inventory);
+    return inventory;
```

Comment:
Before inserting into the database you should make sure you set the id of the inventory object to null.
This is part of the ticket request: "This method should insert a new Inventory object into the Mongo Collection and set the Mongo ID to null."